app/dashboard/templates/profiles/profile.html

```diff
@@ -270,10 +325,89 @@
             $('.profile-bounties--tips .bounty_row').removeClass('bounty_row--hidden');
           });
         }
-        setupTabs('#activity-tabs'); 
+        setupTabs('#activity-tabs');
+
+
+
+        $("[data-toggle=popover]").popover({
+          html : true,
+          trigger: 'focus',
+          template: '<div class="popover wallet-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+          content: function() {
+              var content = $(this).attr("data-popover-content");
+              return $(content).children(".popover-body").html();
+          },
+          title: function() {
+              var title = $(this).attr("data-popover-content");
+              return $(title).children(".popover-heading").html();
+          }
+        });
+
+        {% if not preferred_payout_address%}
+          $("[data-toggle=popover]").popover('show')
+        {% endif %}
 
+        $("body").on('submit', '#form-wallets', function(e) {
+          e.preventDefault()
+          var data = $("#form-wallets").serializeArray();
+          data.push({'page': 2});
+          var postWallets = fetchData ( e.currentTarget.action,
+                                        e.currentTarget.method,
+                                        data
+                                      )
+
+          $.when( postWallets ).then( function ( response ){
+            var walletAddress = response.wallets[0]
+            var newAddress = new truncate(walletAddress)
+            $('#preferred-address').text(newAddress.elem);
+            $('#preferred-address').prop('title', walletAddress)
+            $('#kudos-section').html(response.kudos_html)
+
+            })
+        });
 
       });
+
+
+      (function($) {
+        $(document).on('click','.load-more' ,function() {
+          var address =  $('#preferred-address').prop('title')
+          if (!address) {
+            return
+          }
+          var link = $(this);
+          var page = link.data('page');
+          var request = link.data('request')
+          $.ajax({
+            type: 'POST',
+            url: '/lazy_load_kudos/',
```

Comment:
Since this is a django template, can we replace any hard coded urls like this with `{% url "derp" %}`